druid-shell/src/platform/gtk/window.rs

```diff
@@ -169,12 +172,13 @@ impl WindowBuilder {
         window.set_resizable(self.resizable);
         window.set_decorated(self.show_titlebar);
 
-        // Get the GTK reported DPI
-        let dpi = window
+        // Get the scale factor based on the GTK reported DPI
+        let scale_factor = window
             .get_display()
             .map(|c| c.get_default_screen().get_resolution() as f64)
```

Comment:
Doesn't `get_resolution` already given an `f64`?

Reply:
Indeed it does, good catch!